crates/type-system/src/ontology/uri/wasm.rs

```diff
@@ -0,0 +1,18 @@
+use std::str::FromStr;
+use wasm_bindgen::prelude::*;
+
+use crate::ontology::uri::{error::ParseBaseUriError, BaseUri, VersionedUri};
+use crate::uri::error::ParseVersionedUriError;
+
+#[wasm_bindgen(js_name = parseBaseUri)]
+pub fn parse_base_uri(uri: &str) -> Result<String, ParseBaseUriError> {
+    let url = BaseUri::new(uri)?;
+    Ok(url.to_string())
+}
+
+#[wasm_bindgen(js_name = isValidVersionedUri)]
```

Comment:
Is there something like `toCamelCase`?

Reply:
not that I've found https://github.com/rustwasm/wasm-bindgen/issues/1818